features/step_definitions/web_steps.rb

```diff
@@ -33,6 +33,13 @@ def with_scope(locator)
   page.driver.browser.url_whitelist = ['http://127.0.0.1']
 end
 
+When /^I type in "([^"]*)" with "([^"]*)"$/ do |selector,contents|
+ find(selector).native.send_keys(contents)
+ #sleep 5
+ #puts page.evaluate_script( 'document.documentElement.innerHTML' )
+ puts find(:css , 'div.autocomplete.dropdown', :wait=> 20 , :text => 'uby' ).text
```

Comment:
For what it's worth, the text that autocomplete generates is actually inside a li inside a ul inside the div:

```
<div class="autocomplete dropdown" style="position: absolute; z-index: 999; display: none;">
  <ul role="listbox" aria-activedescendant="ui-active-menuitem" style="display: block;">
    <li role="option" class="odd"><b>Te</b>sting</li>
```

So I don't think looking for it in the div will work -- I think you'll have to look at the list items themselves with something like `div.autocomplete li`